charts/datahub/templates/datahub-upgrade/datahub-restore-indices-job-template.yml

```diff
@@ -98,15 +98,15 @@ spec:
             {{- with .Values.datahubUpgrade.restoreIndices.extraSidecars }}
               {{- toYaml . | nindent 12 }}
             {{- end }}
-          {{- with .Values.datahubUpgrade.nodeSelector }}
+          {{- with default .Values.datahubUpgrade.nodeSelector .Values.global.nodeSelector }}
           nodeSelector:
             {{- toYaml . | nindent 12 }}
           {{- end }}
           {{- with .Values.datahubUpgrade.affinity }}
           affinity:
             {{- toYaml . | nindent 12 }}
           {{- end }}
-          {{- with .Values.datahubUpgrade.tolerations }}
```

Comment:
This looks a little odd to be defaulting to the specific instead of the global. I would think that the specific one would be the override